client/client.go

```diff
@@ -37,27 +39,30 @@ func someUsefulThings() {
 	f[0] = 10
 	userlib.DebugMsg("UUID as string:%v", f.String())
 
-	// takes a sequence of bytes and renders as hex
+	// Takes a sequence of bytes and renders as hex
 	h := hex.EncodeToString([]byte("fubar"))
 	userlib.DebugMsg("The hex: %v", h)
 
 	// Marshals data into a JSON representation
-	// Will actually work with go structures as well
+	// Works well with Go structures!
 	d, _ := userlib.Marshal(f)
 	userlib.DebugMsg("The json data: %v", string(d))
 	var g userlib.UUID
 	userlib.Unmarshal(d, &g)
 	userlib.DebugMsg("Unmashaled data %v", g.String())
 
-	// This creates an error type
+	// errors.New(...) creates an error type!
 	userlib.DebugMsg("Creation of error %v", errors.New(strings.ToTitle("This is an error")))
 
-	// And a random RSA key.  In this case, ignoring the error
-	// return value
+	// Generate a random private/public keypair.
+	// The "_" indicates that we don't check for the error case here.
 	var pk userlib.PKEEncKey
 	var sk userlib.PKEDecKey
 	pk, sk, _ = userlib.PKEKeyGen()
 	userlib.DebugMsg("Key is %v, %v", pk, sk)
+
+	// Useful for string interpolation.
```

Comment:
Can this comment be expanded to explain this in more detail? What does this call to `fmt` accomplish?

Reply:
`fmt.Sprintf` is what I was going for